app/gui/view/component-browser/component-list-panel/breadcrumbs/src/lib.rs

```diff
@@ -431,7 +439,9 @@ ensogl_core::define_endpoints_2! {
         /// Add a new breadcrumb after the currently selected one.
         push(Breadcrumb),
         /// Set the displayed breadcrumbs starting from the specific index.
-        set_entries_from((Vec<Breadcrumb>, usize)),
+        set_entries_from((Vec<Breadcrumb>, BreadcrumbId)),
+        /// Set the breadcrumb at a specified index.
+        set_entry((Breadcrumb, BreadcrumbId)),
```

Comment:
For me, it's more natural to have index as first argument. It's then similar to map's `insert` method (where key goes first).

Reply:
As a matter of preference, I aimed to make it consistent with `set_entries_from`. Changed it, though.